managed/resource_integration.go

```diff
@@ -27,7 +27,9 @@ func (r resourceIntegrationType) GetSchema(_ context.Context) (tfsdk.Schema, dia
 		Description: `The resource to create an export configuration for the integration you want to use in YugabyteDB Aeon.
 
 Using the configuration you created, connect to your cluster:
-	-  Export metrics using resource ` + "`ybm_associate_metrics_exporter_cluster`",
+	-  Export metrics using resource ` + "`ybm_associate_metrics_exporter_cluster`" + `
```

Comment:
Would suggest leaving out changes to this for now, there's some inconsistencies, and formatting issues I would like to address in a separate PR only.